src/schedule-and-details/index.jsx

```diff
@@ -184,6 +185,19 @@ const ScheduleAndDetails = ({ intl, courseId }) => {
               messages.alertSuccessAriaDescribedby,
             )}
           />
+          <AlertMessage
+            show={showFailedAlert}
+            variant="danger"
+            icon={CheckCircleIcon}
```

Comment:
```suggestion
            icon={ErrorOutline}
```
You will also need to import it